subprojects/instant-execution/src/integTest/resources/org/gradle/instantexecution/builds/android-3.6-mini/build.gradle

```diff
@@ -2,12 +2,11 @@
 
 buildscript {
     repositories {
-        maven { url = "https://repo.gradle.org/gradle/ext-snapshots-local/" }
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.0-20190820155739+0200'
+        classpath 'com.android.tools.build:gradle:3.6.0-alpha6'
```

Comment:
❓ Why not the same version as AGP_VERSION?

Reply:
That's what Android Studio generated, so we know what Studio version was used.
Then the test inject the nightly version of AGP.